jsm.go

```diff
@@ -102,22 +105,45 @@ type apiPagedRequest struct {
 	Offset int `json:"offset"`
 }
 
-// accountStats returns current statistics about the account's JetStream usage.
-type accountStats struct {
-	Memory  uint64 `json:"memory"`
-	Store   uint64 `json:"storage"`
-	Streams int    `json:"streams"`
-	Limits  struct {
-		MaxMemory    int64 `json:"max_memory"`
-		MaxStore     int64 `json:"max_storage"`
-		MaxStreams   int   `json:"max_streams"`
-		MaxConsumers int   `json:"max_consumers"`
-	} `json:"limits"`
+// AccountInfo contains info about the JetStream usage from the current account.
+type AccountInfo struct {
+	Memory  uint64        `json:"memory"`
+	Store   uint64        `json:"storage"`
+	Streams int           `json:"streams"`
+	Limits  AccountLimits `json:"limits"`
+}
+
+// AccountLimits includes the JetStream limits of the current account.
+type AccountLimits struct {
+	MaxMemory    int64 `json:"max_memory"`
+	MaxStore     int64 `json:"max_storage"`
+	MaxStreams   int   `json:"max_streams"`
+	MaxConsumers int   `json:"max_consumers"`
 }
 
 type accountInfoResponse struct {
 	apiResponse
-	accountStats
+	AccountInfo
+}
+
+// AccountInfo retrieves info about the JetStream usage from the current account.
+func (js *js) AccountInfo() (*AccountInfo, error) {
+	resp, err := js.nc.Request(js.apiSubj(apiAccountInfo), nil, js.wait)
+	if err != nil {
+		if err == ErrNoResponders {
+			err = ErrJetStreamNotEnabled
+		}
+		return nil, err
+	}
```

Comment:
Should we overwrite `ErrNoResponders` with `ErrJetStreamNotEnabled` here or should we do that in the `JetStream` construction function above?

https://github.com/nats-io/nats.go/pull/639/files#diff-176a2342c55218b70f165bed6a7731dc83d901f9513382f3c98bbd3d08ce0678R134?

I'm thinking about the case where someone calls `AccountInfo` after the constructor function. Not sure which would be more informative. Just a thought.

Reply:
Good point, will add it back to be done when calling `JetStream()` instead.

Reply:
done